api/provider/aws/environment_delete.go

```diff
@@ -53,6 +59,43 @@ func (e *EnvironmentProvider) Delete(environmentID string) error {
 	return nil
 }
 
+func (e *EnvironmentProvider) checkEnvironmentDependencies(environmentID string) error {
+	loadBalancerTags, err := e.TagStore.SelectByType("load_balancer")
+	if err != nil {
+		return err
+	}
+
+	dependentLoadBalancers := loadBalancerTags.WithKey("environment_id").WithValue(environmentID)
+	if len(dependentLoadBalancers) > 0 {
+		msg := fmt.Sprintf("Cannot delete non-empty environment '%s' (environment contains one or more load balancers).", environmentID)
```

Comment:
For each of these errors, we should add the id of the dependent entity. 
Also, the `msg` line is too long, we should break it down into multiple lines.

Reply:
✅